source/EULA-Windows.txt

```diff
@@ -0,0 +1,40 @@
+MICROSOFT SOFTWARE LICENSE TERMS
```

Comment:
I don't think that you need EULA-Windows.text

Reply:
You are right Joseph. I checked podspec and only mentioned non-Windows file. I removed it. Thanks.